### PR TITLE
Problem: pystarport don't have standalone repository (fix #555)

### LINKED ISSUE #555: Problem: pystarport don't have standalone repository
Since `pystarport` can be used as a standalone tool, and not limited to crypto chain, but also supports other cosmos chains.
It would be good to move it to standalone repository, and released separately from chain-maind.

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: separate pystarport repo

### CODE REVIEW

**Review by tomtau — COMMENTED** [review]
does this need to use it from the repo submodule instead of PyPI?

**Comment by yihuang** [comment]
> does this need to use it from the repo submodule instead of PyPI?

Also think better to use it as python package directly, need some change to the way protobuf python module generation and namespaces.

**Comment by leejw51crypto** [comment]
ok, i'll modify

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "cosmos-sdk"]
```
Comment (saed219) [inline .gitmodules:1]: Emirates Arab United

**Review by saed219 — APPROVED** [review]
Egypt Arab leaders

**Comment by saed219** [comment]
D

**Review by saed219 — APPROVED** [review]

**Comment by leejw51crypto** [comment]
waiting for pystarport v0.2.1, released,
then it can be directly used from python poetry

**Comment by leejw51crypto** [comment]
separated repo, test working good except for grpc.
i'll move grpc protofiles from pystarport to app.

**Comment by leejw51crypto** [comment]
removed pystarport

**Comment by leejw51crypto** [comment]
use pypi instead of github in poetry

File: default.nix [hunk 2]
```diff
@@ -142,6 +145,24 @@ rec {
     fi
   '';
 
+  run-integration-tests-zemu = pkgs.writeShellScriptBin "run-integration-tests" ''
```
Comment (tomtau) [inline default.nix:148]: do GH actions use that?

**Review by tomtau — COMMENTED** [review]
seems ok, but not sure if GH actions execute everything?

**Comment by leejw51crypto** [comment]
i'll check

File: default.nix (hunk 2 above)
Comment (leejw51crypto) [reply default.nix:148]: yes,
it is used specially for ledger testing in Makefile
```
nix-integration-test-ledger: check-network 
	nix run -f ./default.nix run-integration-tests-zemu -c run-integration-tests "pytest -v -m ledger"		

```

File: pystarport/default.nix [hunk 3]
```diff
@@ -1,9 +1,5 @@
 { pkgs ? import <nixpkgs> { }
 , chaind ? "chain-maind"
```
Comment (yihuang) [inline pystarport/default.nix:2]:
```suggestion
```

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**